src/app/search/_components/Title.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames/bind';
+import styles from './Title.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface TitleProps {
+  keyword: string;
+  count: number;
+}
+
+export default function Title({ keyword, count }: TitleProps) {
+  return (
+    <div className={cn('title')}>
+      &apos;<div className={cn('keyword')}>{keyword}</div>&apos;&nbsp;
+      <div className={cn('text')}>{`검색 결과(${count <= 99 ? count : '99+'})`}</div>
+    </div>
```

Comment:
```suggestion
    <div className={cn('title')}>
      <div className={cn('text')}>
        {`'${keyword}' 검색 결과(${count <= 99 ? count : '99+'})`}
      </div>
    </div>
```
템플릿 리터럴로 작성하는게 더 가독성이 좋다고 생각합니다~